modules/manage/pages/manage-security/configure-server-certificates.adoc

```diff
@@ -604,23 +618,82 @@ sudo cp ca.pem /opt/couchbase/var/lib/couchbase/inbox/CA/.
 . Upload the root certificate, thereby activating it for the entire cluster.
 +
 ----
-curl -X POST http://10.144.220.103:8091/node/controller/loadTrustedCAs -u Administrator:password
+curl -X POST http://10.143.192.102:8091/node/controller/loadTrustedCAs -u Administrator:password
+----
+
+. Upload the node certificate, specifying the established password for the private key.
```

Comment:
Are we providing instructions for generating **encrypted** node private key here? My understanding is that we are not.
Probably we should not use password here is well but can have a link to that page for reference though.

Reply:
Thanks. I have removed the '-d' and put in the following link.

=== Using an Encrypted, Private Node-Key

The above examples use an _unencrypted_ private key, for the node.
If an _encrypted_ private node-key is used, a passphrase must be registered for it, so that the key can be securely retrieved and used when required.
See the reference page xref:rest-api/upload-retrieve-node-cert.adoc[Upload and Retrieve a Node Certificate].